samples/apps/gradio_gui/gradio_service.py

```diff
@@ -0,0 +1,766 @@
+                                switchy_bt = gr.Button(
+                                    r"Please select from the plugin list first", variant="secondary"
+                                ).style(size="sm")
+                    with gr.Row():
+                        with gr.Accordion(
+                            "Click to expand the `file upload area`。Upload local files/compressed packages for function plugin calls。",
+                            open=False,
+                        ):
+                            file_upload = gr.Files(
+                                label="Any file, Recommend Uploading Compressed File(zip, tar)",
+                                file_count="multiple",
+                                elem_id="elem_upload",
+                            )
+
+        with gr.Floating(init_x="0%", init_y="0%", visible=True, width=None, drag="forbidden"):
+            with gr.Row():
+                with gr.Tab("Upload file", elem_id="interact-panel"):
+                    gr.Markdown(
+                        "Please upload local files/zip packages for `Function Plugin Area` function call。Please note: After uploading the file, the input area will be automatically modified to the corresponding path。"
+                    )
+                    file_upload_2 = gr.Files(
+                        label="Any file, Recommend Uploading Compressed File(zip, tar)", file_count="multiple"
+                    )
+
+                with gr.Tab("Change model & Prompt", elem_id="interact-panel"):
+                    md_dropdown = gr.Dropdown(
+                        AVAIL_LLM_MODELS, value=LLM_MODEL, label="Change LLM model/request source"
+                    ).style(container=False)
+                    top_p = gr.Slider(
+                        minimum=-0,
+                        maximum=1.0,
+                        value=1.0,
+                        step=0.01,
+                        interactive=True,
+                        label="Top-p (nucleus sampling)",
+                    )
+                    temperature = gr.Slider(
+                        minimum=-0,
+                        maximum=2.0,
+                        value=1.0,
+                        step=0.01,
+                        interactive=True,
+                        label="Temperature",
+                    )
+                    max_length_sl = gr.Slider(
+                        minimum=256,
+                        maximum=1024 * 32,
+                        value=4096,
+                        step=128,
+                        interactive=True,
+                        label="Local LLM MaxLength",
+                    )
+                    system_prompt = gr.Textbox(
+                        show_label=True,
+                        lines=2,
+                        placeholder="System Prompt",
+                        label="System prompt",
+                        value=initial_prompt,
+                    )
```

Comment:
Will these values be passed into the agent configuration? I see them only fed into a function imported from `void_terminal`.
![image](https://github.com/microsoft/autogen/assets/3197038/4d5088ea-7163-4548-b0ef-946b8c2166a5)